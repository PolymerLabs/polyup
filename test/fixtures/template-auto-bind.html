<!DOCTYPE html>
<html>
<head>
  <title>Template bind</title>
  <script src='components/webcomponentsjs/webcomponents.js'></script>
  <link rel='import' href='components/polymer/polymer.html'>
</head>
<body>

<template is='auto-binding'>
  <input autofocus value='{{greeting}}'>
  <div>
    Hello world.
  </div>
  <div>
    User says: {{greeting}}
  </div>
  <div>
    This won't work in 1.0: {{greeting + " world"}}
  </div>
</template>

</body>
</html>
