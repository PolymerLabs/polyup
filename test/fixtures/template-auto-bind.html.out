<!DOCTYPE html><html><head>
  <title>Template bind</title>
  <script src="components/webcomponentsjs/webcomponents-lite.js"></script>
  <link rel="import" href="components/polymer/polymer.html">
</head>
<body>

<template is="dom-bind">
  <input autofocus="" value="{{greeting::input}}">
  <div>
    Hello world.
  </div>
  <div>
    User says: <span>{{greeting}}</span>
  </div>
  <!--
      The expression {{greeting + " world"}} can't work in a
      dom-bind template, as it should be an anonymous computed property.
      If you convert it into a Polymer element then polyup should be able to
      upgrade it.
   -->
  <div>
    This won't work in 1.0: <span>{{greeting + " world"}}</span>
  </div>
</template>



</body></html>
